mne/time_frequency/tests/test_psd.py

```diff
@@ -14,99 +16,98 @@
 
 @requires_version('scipy', '0.12')
 def test_psd():
-    """Test PSD estimation
+    """Tests the welch and multitaper PSD
     """
     raw = io.Raw(raw_fname)
-
     exclude = raw.info['bads'] + ['MEG 2443', 'EEG 053']  # bads + 2 more
-
-    # picks MEG gradiometers
-    picks = pick_types(raw.info, meg='mag', eeg=False, stim=False,
-                       exclude=exclude)
-
-    picks = picks[:2]
+    include = []
 
     tmin, tmax = 0, 10  # use the first 60s of data
     fmin, fmax = 2, 70  # look at frequencies between 5 and 70Hz
-
     n_fft = 128
-    psds, freqs = compute_raw_psd(raw, tmin=tmin, tmax=tmax, fmin=fmin,
-                                  fmax=fmax, proj=False, n_fft=n_fft,
-                                  picks=picks, n_jobs=1)
-    assert_true(psds.shape == (len(picks), len(freqs)))
-    assert_true(np.sum(freqs < 0) == 0)
-    assert_true(np.sum(psds < 0) == 0)
-
-    n_fft = 2048  # the FFT size (n_fft). Ideally a power of 2
-    psds, freqs = compute_raw_psd(raw, tmin=tmin, tmax=tmax, picks=picks,
-                                  fmin=fmin, fmax=fmax, n_fft=n_fft, n_jobs=1,
-                                  proj=False)
-    psds_proj, freqs = compute_raw_psd(raw, tmin=tmin, tmax=tmax, picks=picks,
-                                       fmin=fmin, fmax=fmax, n_fft=n_fft,
-                                       n_jobs=1, proj=True)
-
-    assert_array_almost_equal(psds, psds_proj)
-    assert_true(psds.shape == (len(picks), len(freqs)))
-    assert_true(np.sum(freqs < 0) == 0)
-    assert_true(np.sum(psds < 0) == 0)
-
-
-@requires_version('scipy', '0.12')
-def test_psd_epochs():
-    """Test PSD estimation on epochs
-    """
-    raw = io.Raw(raw_fname)
-
-    exclude = raw.info['bads'] + ['MEG 2443', 'EEG 053']  # bads + 2 more
-
-    # picks MEG gradiometers
-    picks = pick_types(raw.info, meg='mag', eeg=False, stim=False,
-                       exclude=exclude)
-
-    picks = picks[:2]
-
-    n_fft = 512  # the FFT size (n_fft). Ideally a power of 2
-
-    tmin, tmax, event_id = -0.5, 0.5, 1
-    include = []
-    raw.info['bads'] += ['MEG 2443']  # bads
 
     # picks MEG gradiometers
     picks = pick_types(raw.info, meg='grad', eeg=False, eog=True,
-                       stim=False, include=include, exclude='bads')
-
+                       stim=False, include=include, exclude=exclude)
+
+    # -- Raw --
+    picks_mt = picks[:2]
+    picks_wel = picks
+    kws_psd = dict(tmin=tmin, tmax=tmax, fmin=fmin, fmax=fmax)
+    kws_welch = dict(picks=picks_wel, n_fft=n_fft)
+    kws_mt = dict(picks=picks_mt, low_bias=True)
+    kws_comp = dict(picks=picks_wel, n_fft=n_fft)
+    funcs = {psd_welch: kws_welch,
+             psd_multitaper: kws_mt,
+             compute_raw_psd: kws_comp}
+
+    with warnings.catch_warnings(record=True):
+        warnings.simplefilter('always')
+        for func, kws in funcs.iteritems():
+            kws = kws.copy()
+            kws.update(kws_psd)
```

Comment:
maybe just add a comment explaining that these arguments are common to all the methods